docs/site/tutorials/todo-list/todo-list-tutorial-has-one-relation.md

```diff
@@ -103,6 +125,45 @@ $ lb4 relation
 Relation BelongsTo was created in src/
 ```
 
+As you can see, this created the relational controllers
```

Comment:
Following the same idea that is used in the todo/id/todlist controller. Move the only method to the todo controller, and delete the relation controller that only has one method. We'll do the same thing here for the /todo-list-image/id/todo-list  method.